tslint.json

```diff
@@ -6,6 +6,7 @@
     "quotemark": [true, "single", "avoid-escape"],
     "member-ordering": false,
     "no-string-based-set-timeout": false,
-    "semicolon": [true, "always", "ignore-bound-class-methods"]
+    "semicolon": [true, "always", "ignore-bound-class-methods"],
+    "no-floating-promises": false
```

Comment:
The new version of @salesforce/dev-config has `"no-floating-promises": true`. I opened the work item W-5676815 to refactor our code to conform to this rule.